### ISSUE TITLE
Electron-cash cannot verify transaction and always see coinbase transaction as unmatured

### ISSUE THREAD

**Issue by jplix7k** [Electron-Cash/Electron-Cash#1914]
We have a wallet for receiving lots of coinbase transaction. Electron-cash could not verify transactions from a certain time and started to always treat them as unmatured coins. 

![image](https://user-images.githubusercontent.com/51077262/87901177-4b436c80-ca91-11ea-98d0-e41a913724d9.png)

Confirmed transaction but could not be verified by Electron-cash:
![image](https://user-images.githubusercontent.com/51077262/87901250-88a7fa00-ca91-11ea-9bfb-0c1ad281e88c.png)

![image](https://user-images.githubusercontent.com/51077262/87901295-9fe6e780-ca91-11ea-9f01-6d6b701878d3.png)

**Comment by jplix7k** [comment]
Get it solved by deleting data and restoring from scratch.

**Comment by cculianu** [comment]
Yeah there's bug if Electron Cash sees "chain forks" throughout its operation.  I have to fix it.  Sorry about that.  One of the ways to fix it is indeed as you have done -- rebuild all data from scratch. The other way is to got electron cash's own data folder and delete the `forks` directory (do this with Electron Cash stopped).  It's a known issue that needs fixing.  Glad you figured it out.
